### PR TITLE
Adding onnxconverter-common

### PR DESCRIPTION
**Pull request by DavidKatz-il**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dopplershift** [comment]
@sephib Are you willing to be a maintainer on this recipe?

File: recipes/onnxconverter-common/meta.yaml
Comment (synapticarbors) [inline recipes/onnxconverter-common/meta.yaml:18]: You can remove the empty `build` section

File: recipes/onnxconverter-common/meta.yaml
Comment (synapticarbors) [inline recipes/onnxconverter-common/meta.yaml:30]: This isn't testing the package; it's just importing `unittest`. You should replace `unittest` with `onnxconverter-common`.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/onnxconverter-common/meta.yaml
Comment (synapticarbors) [reply recipes/onnxconverter-common/meta.yaml:29]: It translates the item in the yaml list to basically a python script that calls `import x`, where `x` is the item in the list. I'm assuming you import the package using `onnxconverter_common` since you can't have dashes in an import name.

Comment (DavidKatz-il) [reply recipes/onnxconverter-common/meta.yaml:29]: Thanks, Got it.

**Comment by synapticarbors** [comment]
Everything looks good now. We're just waiting on the ok from the listed co-maintainer to say they are willing to be listed.

**Comment by sephib** [comment]
Hi, 
I'm waiting for an agreement from the onnxconverter  team

**Pull request closed without merging**
